sdcm/nemesis.py

```diff
@@ -4670,6 +4694,7 @@ class RejectInterNodeNetworkMonkey(Nemesis):
     disruptive = True
     networking = True
     run_with_gemini = False
+    free_tier_set = True
```

Comment:
This nemesis is absent in the list you posted here: https://github.com/scylladb/scylla-cluster-tests/pull/5504#issuecomment-1337171769
Which one is correct?